package.json

```
@@ -732,34 +732,42 @@
-		"build:web:esbuild": "node esbuild.mjs --mode web",
+		"build:web:esbuild": "node ./scripts/esbuild.mjs --mode web",
 		"build:desktop:tsc": "tsc --noEmit --project tsconfig.app.json",
-		"build:desktop:esbuild": "node esbuild.mjs --mode desktop",
+		"build:desktop:esbuild": "node ./scripts/esbuild.mjs --mode desktop",
 		"watch:web": "npm-run-all -p watch:web:*",
 		"watch:desktop": "npm-run-all -p watch:desktop:*",
-		"watch:web:esbuild": "node esbuild.mjs --watch --mode web",
+		"watch:web:esbuild": "node ./scripts/esbuild.mjs --watch --mode web",
 		"watch:web:tsc": "tsc --noEmit --watch --project tsconfig.web.json",
-		"watch:desktop:esbuild": "node esbuild.mjs --watch --mode desktop",
+		"watch:desktop:esbuild": "node ./scripts/esbuild.mjs --watch --mode desktop",
 		"watch:desktop:tsc": "tsc --noEmit --watch --project tsconfig.app.json",
-		"test:web": "npm run build:web && npm-run-all -s test:web:*",
-		"test:desktop": "npm run build:desktop && npm-run-all -s test:desktop:*",
+		"test:web": "pnpm run build:web && npm-run-all -s test:web:*",
 		"test:web:tsc": "tsc --noEmit --project test-tsconfigs/tsconfig.web.json",
-		"test:web:esbuild": "node esbuild.mjs --mode web --test",
-		"test:web:vscode": "vscode-test --coverage --label webUnitTest",
+		"test:web:esbuild": "node ./scripts/esbuild.mjs --mode web --test",
+		"test:web:vscode": "vscode-test --coverage --label webUnitTest --coverage-output ./web-coverage",
+		"test:web:browser": "pnpm run build:web && npm-run-all -s test:web:browser:*",
+		"test:web:browser:tsc": "tsc =--noEmit --project test-tsconfigs/tsconfig.web.json",
+		"test:web:browser:esbuild": "node ./scripts/esbuild.mjs --mode web --test --web-browser-test",
+		"test:web:browser:open": "npm-run-all -s test:web:browser:open:* --continue-on-error",
+		"test:web:browser:open:chromium": "vscode-test-web --headless --browser=chromium --extensionDevelopmentPath=. --extensionTestsPath=out/web/test/browser.js test-playground",
+		"test:web:browser:open:firefox": "vscode-test-web --headless --browser=firefox --extensionDevelopmentPath=. --extensionTestsPath=out/web/test/browser.js test-playground",
+		"test:web:browser:open:webkit": "vscode-test-web --headless --browser=webkit --extensionDevelopmentPath=. --extensionTestsPath=out/web/test/browser.js test-playground",
+		"test:web:browser:open:vscode": "vscode-test-web --headless --extensionDevelopmentPath=. --extensionTestsPath=out/web/test/browser.js test-playground",
+		"test:desktop": "npm run build:desktop && npm-run-all -s test:desktop:*",
 		"test:desktop:tsc": "tsc --noEmit --project test-tsconfigs/tsconfig.app.json",
-		"test:desktop:esbuild": "node esbuild.mjs --mode desktop --test",
-		"test:desktop:vscode": "vscode-test --coverage --label desktopUnitTest"
+		"test:desktop:esbuild": "node ./scripts/esbuild.mjs --mode desktop --test",
+		"test:desktop:vscode": "vscode-test --coverage --label desktopUnitTest --coverage-output ./desktop-coverage"
```

Change name format from `<platform>-coverage` to `coverage-<platform>` so they are all next to each other in the file browser.
```suggestion
		"test:desktop:vscode": "vscode-test --coverage --label desktopUnitTest --coverage-output ./coverage-desktop"
```